src/AmbientSounds.Uwp/Controls/SettingsControl.xaml

```diff
@@ -20,6 +37,15 @@
                     <TextBlock x:Uid="OnTextBlock" />
                 </ToggleSwitch.OnContent>
             </ToggleSwitch>
+
+            <TextBlock x:Uid="AppThemeTextBlock" Text="Application theme" Style="{StaticResource SettingTitleStyle}" />
+            <RadioButton x:Uid="DefaultThemeRadio" Content="Use system setting"
+                         Click="{x:Bind ViewModel.DefaultThemeRadioClicked}" IsChecked="{x:Bind ViewModel.DefaultRadioIsChecked}" />
+            <RadioButton x:Uid="DarkThemeRadio" Content="Dark theme"
+                         Click="{x:Bind ViewModel.DarkThemeRadioClicked}" IsChecked="{x:Bind ViewModel.DarkRadioIsChecked}" />
+            <RadioButton x:Uid="LightThemeRadio" Content="Light theme"
+                         Click="{x:Bind ViewModel.LightThemeRadioClicked}" IsChecked="{x:Bind ViewModel.LightRadioIsChecked}" />
+            <TextBlock x:Uid="ThemeWarningTextBlock" Text="Theme will be changed after restart." Name="ThemeWarning" Style="{StaticResource ThemeWarningTextStyle}" />
```

Comment:
I forgot to mention this in the readme, but all xaml in Ambie is auto styled using this extension: https://marketplace.visualstudio.com/items?itemName=TeamXavalon.XAMLStyler. When you install that extension, pressing save while viewing a xaml file will autoformat the xaml. Can you try using this auto formatter for all the xaml files edited in this pr?